hw/dv/sv/sec_cm/sec_cm.core

```diff
@@ -0,0 +1,23 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:sec_cm"
```

Comment:
`sec_cm_bind`

Reply:
this core file contains bind, interface, pkg. sec_cm may be better